src/components/notification/Notification.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
```

Comment:
Just for consistency with the rest of the code base, let's import the prop types like this:

```
import { oneOf, string, ..... } from 'prop-types';
```